lib/Dialect/FIRRTL/FIRRTLUtils.cpp

```diff
@@ -106,7 +106,6 @@ void circt::firrtl::emitConnect(ImplicitLocOpBuilder &builder, Value dst,
     // Const-cast as needed, using widthless version of dest.
     // (dest is either widthless already, or source is and if the types
     //  can be const-cast'd, do so)
-    assert(srcType.isGround() && dstType.isGround());
```

Comment:
Line 99 above refers to `ground` types, I suppose that can be removed or changed to `passive`

Reply:
Good call, thank you!